### ISSUE TITLE
validation set

### ISSUE THREAD

**Issue by kaishxu** [dvlab-research/Step-DPO#12]
Hello,

I found the dataset in HF only contain training samples, and there is only a test set in this repo. Could you please introduce how do you obtain the validation set?
